config/install/acquia_lift.settings.yml

```diff
@@ -22,4 +22,5 @@ visibility:
   path_patterns: "/admin\n/admin/*\n/batch\n/node/add*\n/node/*/*\n/user/*\n/block/*"
 advanced:
   content_replacement_mode: 'untrusted'
+  bootstrap_mode: 'auto'
```

Comment:
Please rotate line 24 and 25 to keep the same order between bootstrap_mode and content_replacement_mode across the rest of the applications.

Reply:
Should be a matching setting in https://github.com/acquia/acquia_lift/blob/8d7a035420073c2350b3eb4a7beb045e08dfb2fa/config/schema/acquia_lift.settings.yml#L115